src/renderer/containers/Validator/Validator.tsx

```diff
@@ -65,14 +57,16 @@ export const Validator: React.FunctionComponent<IValidatorSimpleProps> = (props:
                                     onClick={props.onBeaconNodeClick(node.url)}
                                     title={node.localDockerId ? "Local Docker container" : "Remote Beacon node"}
                                     url={node.url}
-                                    isSyncing={node.isSyncing}
-                                    value={node.currentSlot || "N/A"}
+                                    isSyncing={false}
```

Comment:
?

Reply:
hardcoded for the first stage of refactoring, by design value should be ping but by code is a slot
and probably `isSyncing` should be `status` whit `enum` of `offline` `online` `syncing`
We gonna need to discuss this in detail at the meeting.